toolchain/sem_ir/facet_type_info.h

```diff
@@ -63,6 +64,17 @@ struct FacetTypeInfo : Printable<FacetTypeInfo> {
   // These are the required interfaces that are not lookup contexts.
   llvm::SmallVector<ImplsConstraint> self_impls_constraints;
 
+  // These are the named constraints required by the facet type. The interfaces
+  // required by the named constraint are inlined into the other constraint
+  // types by Identifying the facet type, so this is always empty for an
+  // Identified facet type.
```

Comment:
Why is "Identifying"/"Identified" capitalized? Maybe one of:
```suggestion
  // types by identifying the facet type, so this is always empty for an
  // identified facet type.
```
```suggestion
  // types by identifying the facet type, so this is always empty for an
  // IdentifiedFacetType.
```

Reply:
Actually by the time I was done I saw we have IdentifiedFacetType as a type already, so I will remove this.